spec/features/user_sees_contacts_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "User sees contacts" do
```

Comment:
What benefit does this test provide? You have these same things covered in other specs.